@here/harp-mapview/lib/MapView.ts

```diff
@@ -2704,7 +2708,7 @@ export class MapView extends THREE.EventDispatcher {
 
         // Make sure to update all properties that are accessable via API (e.g. zoomlevel) b/c
         // otherwise they would be updated as recently as in the next animation frame.
-        this.updateLookAtSettings();
+        this.updateViewState();
```

Comment:
When `mapView.lookAt()` is called target, zoom level, pitch and heading are used to compute targetDistance, camera position and camera attitude. But `mapView.updateViewState()` in turn will compute target, targetDistance and zoom level again from the camera position and attitude. So the value of `mapView.target` and `mapView.zoomLevel` can be slightly different than the value set with `mapView.lookAt({ target, zoomLevel })`. I understand that this is necessary if the camera position and attitude was directly modified from outside of MapView but it seems confusing if a user only uses the `mapView.lookAt()` API to change camera position. Maybe it would be better if users that directly manipulate `mapView.camera` need to explicitly call `mapView.updateViewState()`(or something similar) to update target and zoom level from the camera position.